src/nft/operations.cc

```diff
@@ -726,35 +726,46 @@ Nft mata::nft::invert_levels(const Nft& aut, const JumpMode jump_mode) {
     // Creates inverted transitions for each transition in the path.
     // Can work with different jump modes.
     auto map_inverted_transitions = [&](const std::vector<Transition>& path, const State head, const State tail) {
-        for (const auto &[src, symbol, trg]: path) {
+        // Auxiliary state between two states can be reused for transitions over different symbols.
+        // The key is a pair of source and target states.
+        // auxiliary_states map will be used only if the jump_mode is JumpMode::AppendDontCares.
+        std::unordered_map<std::pair<State, State>, State> auxiliary_states;
+        auto get_aux_state = [&](const State src, const State tgt, const size_t level) {
```

Comment:
I would use `Level level` here. As a bonus, this removes the need for the `static_cast<>()`. And `static_cast<>()`, if necessary, can be used at the point of calling the function.